submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,161 @@
+const NUM_OF_FRIENDS = 20;
+const FRIENDS = document.querySelector(".friends");
+let allFriends;
+let friendsForFilter;
+let isFiltered = false;
+let justMale;
+let justFemale;
+
+function fetchFriends() {
+  const api_url = `https://randomuser.me/api/?results=${NUM_OF_FRIENDS}`;
+  fetch(api_url)
+    .then(handleErrors)
+    .then((response) => response.json())
+    .then((data) => (allFriends = data.results))
+    .then(() => addFriends(allFriends))
+    .catch((error) => showErrorMessage());
+}
+
+function handleErrors(response) {
+  if (!response.ok) {
+    throw Error(response.statusText);
+  }
+  return response;
+}
+
+function showErrorMessage() {
+  const wholePage = document.querySelector("body");
+  wholePage.setAttribute("class", "refresh");
+  wholePage.innerHTML = `<div>Ops...Something went wrong.</div>
+        <div>Refresh the page to start finding friends</div>`;
```

Comment:
Once again - why do you need a refresh? What the reason?

Reply:
Just do something to simplify users lives)).  What if they see the error message and don't understand what to do next. And I give them a tip. I don't understand, why it is wrong - to show some advise to users.

Reply:
Removed a tip.